<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite">
    <groups>
      <group name="run">
        <method signature="org.scalatest.tools.test.SuiteWithSkippedTest.dependeeThatFails()" name="dependeeThatFails" class="org.scalatest.tools.test.SuiteWithSkippedTest"/>
      </group>
    </groups>
    <test name="Command line test">
      <class name="org.scalatest.tools.test.SuiteWithSkippedTest">
        <test-method status="SKIP" signature="depender()" name="depender" duration-ms="27" depends-on-groups="run" started-at="2016-08-26T11:02:46Z" finished-at="2016-08-26T11:02:46Z">
        </test-method>
        <test-method status="FAIL" signature="dependeeThatFails()" name="dependeeThatFails" duration-ms="75" started-at="2016-08-26T11:02:46Z" finished-at="2016-08-26T11:02:46Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: fail
	at org.scalatest.tools.test.SuiteWithSkippedTest.dependeeThatFails(ScalaTestRunnerSuite.scala:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:580)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:478)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:607)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:874)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:689)
	at org.testng.TestRunner.privateRun(TestRunner.java:566)
	at org.testng.TestRunner.run(TestRunner.java:466)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:301)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:296)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:276)
	at org.testng.SuiteRunner.run(SuiteRunner.java:191)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:808)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:776)
	at org.testng.TestNG.run(TestNG.java:701)
	at org.scalatest.testng.TestNGSuite$class.run(TestNGSuite.scala:161)
	at org.scalatest.tools.test.SuiteWithSkippedTest.run(ScalaTestRunnerSuite.scala:213)
	at org.scalatest.testng.TestNGSuite$class.runTestNG(TestNGSuite.scala:148)
	at org.scalatest.tools.test.SuiteWithSkippedTest.runTestNG(ScalaTestRunnerSuite.scala:213)
	at org.scalatest.testng.TestNGSuite$class.run(TestNGSuite.scala:99)
	at org.scalatest.tools.test.SuiteWithSkippedTest.run(ScalaTestRunnerSuite.scala:213)
	at org.scalatest.tools.ScalaTestFramework$ScalaTestRunner.run(ScalaTestFramework.scala:104)
	at org.scalatest.tools.ScalaTestRunnerSuite.run(ScalaTestRunnerSuite.scala:154)
	at org.scalatest.tools.ScalaTestRunnerSuite.run(ScalaTestRunnerSuite.scala:144)
	at org.scalatest.tools.ScalaTestRunnerSuite$$anonfun$9.apply(ScalaTestRunnerSuite.scala:112)
	at org.scalatest.tools.ScalaTestRunnerSuite$$anonfun$9.apply(ScalaTestRunnerSuite.scala:111)
	at org.scalatest.FunSuite$$anon$2.apply(FunSuite.scala:1146)
	at org.scalatest.Suite$class.withFixture(Suite.scala:1477)
	at org.scalatest.tools.ScalaTestRunnerSuite.withFixture(ScalaTestRunnerSuite.scala:23)
	at org.scalatest.FunSuite$class.runTest(FunSuite.scala:1143)
	at org.scalatest.tools.ScalaTestRunnerSuite.runTest(ScalaTestRunnerSuite.scala:23)
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1252)
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1243)
	at scala.collection.LinearSeqOptimized$class.foreach(LinearSeqOptimized.scala:61)
	at scala.collection.immutable.List.foreach(List.scala:45)
	at org.scalatest.FunSuite$class.runTests(FunSuite.scala:1243)
	at org.scalatest.tools.ScalaTestRunnerSuite.runTests(ScalaTestRunnerSuite.scala:23)
	at org.scalatest.Suite$class.run(Suite.scala:1772)
	at org.scalatest.tools.ScalaTestRunnerSuite.org$scalatest$FunSuite$$super$run(ScalaTestRunnerSuite.scala:23)
	at org.scalatest.FunSuite$class.run(FunSuite.scala:1289)
	at org.scalatest.tools.ScalaTestRunnerSuite.run(ScalaTestRunnerSuite.scala:23)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:59)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
